src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery_mg.md

```diff
@@ -7,6 +7,8 @@ title: Magnifier widget
 Magnifier is a [widget](https://glossary.magento.com/widget) that allows displaying images in 100% scaled size in a separate dedicated layer.
 In Magento it is used by the [gallery]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_gallery.html) widget for zooming the product images on product pages. It must be enabled in your local theme `view.xml` configuration file.
 
+The Magnifier widget source is located in [lib/web/magnifier/magnify.js].
```

Comment:
Not defined as a link so needs different formatting.
```suggestion
The Magnifier widget source is located in `lib/web/magnifier/magnify.js`.
```

Reply:
Hi @BarnyShergold 
Your suggestion have been implemented!
Could you please verify this PR?
Thank you!